deploy/olm-catalog/operand-deployment-lifecycle-manager/1.2.1/operand-deployment-lifecycle-manager.v1.2.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,569 @@
+          - operand-deployment-lifecycle-manager
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          verbs:
+          - get
+          - list
+        - apiGroups:
+          - apps
+          resources:
+          - replicasets
+          - deployments
+          verbs:
+          - get
+        - apiGroups:
+          - operator.ibm.com
+          - operators.coreos.com
+          resources:
+          - '*'
+          - operandregistries
+          - operandconfigs
+          - operandrequests
+          - operandbindinfos
+          verbs:
+          - '*'
+        - apiGroups:
+          - '*'
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        - nonResourceURLs:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: operand-deployment-lifecycle-manager
+      deployments:
+      - name: operand-deployment-lifecycle-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: operand-deployment-lifecycle-manager
+          strategy: {}
+          template:
+            metadata:
+              annotations:
+                productID: 068a62892a1e4db39641342e592daa25
+                productMetric: FREE
+                productName: IBM Cloud Platform Common Services
+                productVersion: 3.4.1
+              labels:
+                app.kubernetes.io/instance: operand-deployment-lifecycle-manager
+                app.kubernetes.io/managed-by: operand-deployment-lifecycle-manager
+                app.kubernetes.io/name: odlm
+                name: operand-deployment-lifecycle-manager
+            spec:
+              affinity:
+                nodeAffinity:
+                  requiredDuringSchedulingIgnoredDuringExecution:
+                    nodeSelectorTerms:
+                    - matchExpressions:
+                      - key: beta.kubernetes.io/arch
+                        operator: In
+                        values:
+                        - amd64
+                        - ppc64le
+                        - s390x
+              containers:
+              - args:
+                - -v=2
+                command:
+                - odlm
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: OPERATOR_NAME
+                  value: operand-deployment-lifecycle-manager
+                image: quay.io/opencloudio/odlm:1.2.1
```

Comment:
what's the difference between 1.2.1 and latest?

Reply:
1.2.1 is picked by CICD from the latest image. I will update it to use latest